src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -242,7 +242,7 @@ const SubmissionDetails = _.flow(
           })
         ])
       ]),
-      div({ style: { flex: 1 } }, [
+      div({ style: { flex: `1 0 calc(${1 + Math.min(filteredWorkflows.length, 5.5)} * 48px)` } }, [
```

Comment:
This seems to be working pretty well, but we prefer using rems instead of pixels for sizing things based on text size. rems are linked to the root font size, which is 16 in Terra. So 3 rems should be equivalent to 48 px. The advantage to using rems is that it will scale appropriately as the user zooms in and out.

Also, we prefer to use the lodash utility functions, so please use `_.min` instead of `Math.min`.

Reply:
I should have put a comment here, but the 48px is based on the row height of the FlexTable component.
https://github.com/DataBiosphere/terra-ui/blob/85d11a9972f2f1fb09f21a0534204c2983a6da98/src/components/table.js#L210-L214

Reply:
Ah, I see. A comment in the code would be useful, and then you can just do the multiplication in JavaScript and avoid the use of css `calc`.

Reply:
(Sorry to be late to the party 😛)

> 48px is based on the row height of the FlexTable component

I wonder if we should pull out that `48` onto a dedicated `const` and have `FlexTable` and the newly edited code reference it. The concern would be that if `FlexTable` spec changes, it'd be tough to know to also change the other place.

Reply:
A few options. We could...
* Export a constant with the default row height of FlexTable and use it here. However, that could be overridden by setting the row height on the submissions table here.
* Use a row height constant in this file for this and the submissions table row height. However, if the default row height of FlexTable changes, this table would be non-standard.

In any case, the stakes are low. If the FlexTable default changes, the consequence would be that this table doesn't show exactly 5.5 rows in browser windows less than a certain height.

Reply:
To clarify, I was suggesting
- `export`ing a `const` equal to `48` in `table.js`, 
- using the `const` in `FlexTable` definition in `table.js`
- replacing `48` in `SubmissionDetails.js` with the `const`

If someone consciously wants to override the default, they can, which is the current situation and is fine. I just wanted to avoid having to change multiple places when a value changes.

Reply:
To clarify, I was suggesting
- `export`ing a `const` equal to `48` in `table.js`, 
- using the `const` in `FlexTable` definition in `table.js`
- replacing `48` in `SubmissionDetails.js` with the `const`

If someone consciously wants to override the default, they can, which is the current situation and is fine. I just wanted to avoid having to change multiple places when a value changes.